templates/pages/admin/form/form_question.html.twig

```diff
@@ -246,6 +255,11 @@
             name="type"
             value="{{ constant('class', question_type) }}"
         />
+        <input
+            type="hidden"
+            name="allow_anonymous"
+            value="{{ not is_anonymous or question_type.isAllowedForAnonymousForm() ? 1 : 0 }}"
+        />
```

Comment:
Since the value is not saved into the database and is only used for javascript logic, is there a way to store the information in a data attribute instead ?